netlify.toml

```diff
@@ -61,9 +61,3 @@
 
 [build.environment]
   NODE_OPTIONS = "--max_old_space_size=4096"
-
-[[redirects]]
```

Comment:
somehow this redirect directive is not working, it's documented in netlify, but there must be a reason it doesn't work in our setup. i'm really not sure why. but see my other comment for an alternative that should work.